src/main/kotlin/org/partiql/scribe/targets/spark/SparkDialect.kt

```diff
@@ -113,6 +114,15 @@ public open class SparkDialect : SqlDialect() {
         return tail concat list(start, end) { node.values }
     }
 
+    @OptIn(PartiQLValueExperimental::class)
```

Comment:
Spark date and timestamp literals follow same behavior as PartiQL. Spark does not have a time type so error.